app/src/main/java/com/zaffierce/taskplanner/allactivity.java

```diff
@@ -0,0 +1,14 @@
+package com.zaffierce.taskplanner;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+
+public class allactivity extends AppCompatActivity {

```

Comment:
"allactivity" should be in upper camel case like a regular Java class: AllActivity